src/containers/Review/assignment/AssigneeDropdown.tsx

```diff
@@ -1,28 +1,27 @@
 import React from 'react'
 import { Dropdown } from 'semantic-ui-react'
+import { useLanguageProvider } from '../../../contexts/Localisation'
+import { AssignmentOptions } from '../../../utils/types'
 interface AssigneeProps {
-  assignmentOptions: {
-    isCompleted: boolean
-    selected: number
-    options: {
-      key: number
-      value: number
-      text: string
-    }[]
-  }
+  assignmentOptions: AssignmentOptions
   sectionCode: string
   onChangeMethod: (selected: number) => void
 }
 
 const AssigneeDropdown: React.FC<AssigneeProps> = ({ assignmentOptions, onChangeMethod }) => {
+  const { strings } = useLanguageProvider()
   const { isCompleted, options, selected } = assignmentOptions
 
   return (
     <Dropdown
       className="reviewer-dropdown"
+      placeholder={strings.ASSIGNMENT_NOT_ASSIGNED}
       options={options}
       value={selected}
       disabled={isCompleted}
+      labeled
```

Comment:
What exactly is `labeled` doing here? I don't see any difference.

Reply:
Oh, Cool! I think it was used to show the selected option before. I'll remove